apps/voice_rag/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Use the official micromamba image as a base
+FROM docker.io/mambaorg/micromamba:1.5-jammy
+
+# Create a new user '$MAMBA_USER' and set the working directory
+COPY --chown=$MAMBA_USER:$MAMBA_USER environment.docker.yml /tmp/environment.yml
+
+# Install the specified packages using micromamba
+RUN micromamba install -y -n base -f /tmp/environment.yml && \
+    micromamba clean --all --yes    
+
+USER root
+WORKDIR /usr/local/src
+
+ARG VOCODE_USER=vocode
+ARG VOCODE_UID=8476
+ARG VOCODE_GID=8476
+
+RUN groupadd --gid $VOCODE_GID $VOCODE_USER && \
+    useradd --uid $VOCODE_UID --gid $VOCODE_GID --shell /bin/bash --create-home $VOCODE_USER
+
+# COPY --chown=$VOCODE_USER:$VOCODE_USER ../../../ /vocode-python
+# WORKDIR /usr/local/src/vocode
+# RUN poetry install -E all
+
+# Copy the rest of your application files into the Docker image
+COPY --chown=$VOCODE_USER:$VOCODE_USER . /vocode
+WORKDIR /vocode
+
+#USER vocode
+USER root
+
+ENV OPENAI_API_KEY=""
```

Comment:
nit: can we use the .env to populate these?

Reply:
i see below we use `--env-file` anyways, so can probably remove this block?

Reply:
Thanks for your suggestion regarding the environment variables in the Dockerfile. I initially declared them (ENV) as a common practice to document and indicate the variables our application expects. However, I agree with your point about removing them. Given the extensive number of environment variables we consider in our project, it would be more practical and less cluttered to manage these through our .env file instead. This approach aligns well with best practices for maintainability while keeping our Dockerfile clean and efficient.